dashboard/src/actions/repos.ts

```diff
@@ -433,16 +433,8 @@ export function createDockerRegistrySecret(
       clusters: { currentCluster },
     } = getState();
     try {
-      const secret = await Secret.createPullSecret(
-        currentCluster,
-        name,
-        user,
-        password,
-        email,
-        server,
-        namespace,
-      );
-      dispatch(createImagePullSecret(secret));
+      await Secret.createPullSecret(currentCluster, name, user, password, email, server, namespace);
```

Comment:
Not sure if we can/want use the `Context` here instead of passing the cluster/ns directly

Reply:
Yep, we probably can now, but will need to update the form props and call-sites etc.